### PR TITLE
NMA-340 Disabled biometric limit when fingerprint disabled

### PR DESCRIPTION
**Pull request by dnllb**

### CODE REVIEW

File: common/src/main/java/org/dash/wallet/common/Configuration.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
     public static final String PREFS_KEY_AUTO_LOGOUT_ENABLED = "auto_logout_enabled";
     public static final String PREFS_KEY_AUTO_LOGOUT_MINUTES = "auto_logout_minutes";
     private static final String PREFS_KEY_SPENDING_CONFIRMATION_ENABLED = "spending_confirmation_enabled";
-    private static final String PREFS_KEY_SPENDING_CONFIRMATION_LIMIT = "spending_confirmation_limit";
+    private static final String PREFS_KEY_BIOMETRIC_LIMIT = "biometric_limit";
```
Comment (denlb) [inline common/src/main/java/org/dash/wallet/common/Configuration.java]: This wording makes more sense

Comment (HashEngineering) [reply common/src/main/java/org/dash/wallet/common/Configuration.java]: this change is ok.

**Comment by dnlilib** [comment]
@HashEngineering you're right. Preference does not reflect the UI toggle.

**Comment by tomasz-ludek** [comment]
All issues fixed

**Pull request merged**